packages/mds-compliance/api.ts

```diff
@@ -105,16 +118,14 @@ function api(app: express.Express): express.Express {
         policy &&
         ((policy.provider_ids && policy.provider_ids.includes(provider_id)) ||
           !policy.provider_ids ||
-          ([TEST1_PROVIDER_ID, TEST2_PROVIDER_ID, TEST4_PROVIDER_ID].includes(provider_id) &&
+          (AllowedProviderIDs.includes(provider_id) &&
```

Comment:
Not a blocker, but I'm wondering why the auth0 scoping can't be used here?

Reply:
Good question. Ultimately, that's the plan. We can't enable scope enforcement though until all providers switch over to using the new access token audiences.